winui-toc.html

```diff
@@ -375,6 +375,7 @@
         <ul>
             <li> <a href="/winui/Kanban/overview">Overview</a></li>
             <li><a href="/winui/Kanban/getting-started">Getting Started</a></li>
+            <li><a href="/winui/Kanban/swimlane">Swimlane</a></li>
```

Comment:
Follow this order

- Overview
- Getting Started
- Columns
- Cards
- Swimlane

Reply:
Will add once Columns and Cards merged.